Streetcode/Streetcode.WebApi/Program.cs

```diff
@@ -20,6 +20,7 @@
     app.UseSwagger();
     app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "WebAPIv5 v1"));
     await app.ApplyMigrations();
+    await app.SeedDataAsync();
```

Comment:
it means, that every time, that we run the app seeding will be here?

Reply:
Yeah, you can simply comment this line after succesful seeding of data